### ISSUE TITLE
unable to generate *.pb.cr file from spec/fixtures

### ISSUE THREAD

**Issue by ttdonovan** [jeromegn/protobuf.cr#1]
When attempting to generate the `spec/fixtures/test.pb.cr` file a `Nil assertion failed (exception)` is being raised. I'll try to look into this more either later this evening or tomorrow, but wanted to submit an issue just incase I'm doing something obviously wrong with the `protoc -I` command.

```
Pair@dyn-client-144:~/Workspace/ttdonovan/protobuf.cr (master)$ crystal --version
Crystal 0.18.7 (2016-07-03)
Pair@dyn-client-144:~/Workspace/ttdonovan/protobuf.cr (master)$ crystal spec
....

Finished in 974 microseconds
4 examples, 0 failures, 0 errors, 0 pending
Pair@dyn-client-144:~/Workspace/ttdonovan/protobuf.cr (master)$ protoc --version
libprotoc 2.6.1
Pair@dyn-client-144:~/Workspace/ttdonovan/protobuf.cr (master)$ protoc -I spec/fixtures/ --crystal_out spec/fixtures/ spec/fixtures/
Generating protobuf classes... (protobuf.cr v2.0.1)
Nil assertion failed (Exception)
[4479248563] *raise<String>:NoReturn +163
[4479273536] *Nil#not_nil!:NoReturn +16
[4479245966] __crystal_main +13486
[4479294126] main +46
--crystal_out: protoc-gen-crystal: Plugin failed with status code 1.
Pair@dyn-client-144:~/Workspace/ttdonovan/protobuf.cr (master)$ protoc -I spec/fixtures/ --crystal_out spec/fixtures/ spec/fixtures/test.p
test.pb.cr  test.proto
Pair@dyn-client-144:~/Workspace/ttdonovan/protobuf.cr (master)$ protoc -I spec/fixtures/ --crystal_out spec/fixtures/ spec/fixtures/test.proto
Generating protobuf classes... (protobuf.cr v2.0.1)
Nil assertion failed (Exception)
[4328536243] *raise<String>:NoReturn +163
[4328561216] *Nil#not_nil!:NoReturn +16
[4328533646] __crystal_main +13486
[4328581806] main +46
--crystal_out: protoc-gen-crystal: Plugin failed with status code 1.
```

**Comment by jeromegn** [comment]
Hmm. I wrote that test.pb.cr by hand. I'll have to look into that.

It's interesting because I've generated much more complex files from
multiple .proto referencing themselves. You'd think that would cover all
cases :)

Thanks for the report!

**Comment by jeromegn** [comment]
@ttdonovan I believe this is fixed in v2.0.2.

Turns out I was assuming every protobuf file would define a package name.

It still doesn't generate a 100% working .pb.cr for this particular `test.proto` because it uses extensions and I haven't devised a way to work those out at the moment.

I've updated my homebrew tap as well.
